frontend/src/components/util/Toggle/toggle.module.scss

```diff
@@ -0,0 +1,62 @@
+.container {
+  padding-left: 4rem;
+}
+
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 34px;
+}
+
```

Comment:
let's add comment of `for dark theme` **above all the classes** that you have written for dark theme